src/app/public/modules/lookup/lookup.module.ts

```diff
@@ -1,25 +1,79 @@
-import { NgModule } from '@angular/core';
-import { CommonModule } from '@angular/common';
-import { FormsModule } from '@angular/forms';
+import {
+  NgModule
+} from '@angular/core';
+
+import {
+  CommonModule
+} from '@angular/common';
 
 import {
-  SkyAppWindowRef
+  FormsModule
+} from '@angular/forms';
+
+import {
+  SkyAppWindowRef,
+  SkyViewkeeperModule
 } from '@skyux/core';
 
-import { SkyAutocompleteModule } from '../autocomplete/autocomplete.module';
-import { SkyTokensModule } from '@skyux/indicators';
+import {
+  SkyDataManagerModule
+} from '@skyux/data-manager';
+
+import {
+  SkyIconModule,
+  SkyTokensModule
+} from '@skyux/indicators';
+
+import {
+  SkyInfiniteScrollModule,
+  SkyRepeaterModule
+} from '@skyux/lists';
 
-import { SkyLookupComponent } from './lookup.component';
+import {
+  SkyModalModule
+} from '@skyux/modals';
+
+import {
+  SkyThemeModule
+} from '@skyux/theme';
+
+import {
+  SkyLookupComponent
+} from './lookup.component';
+
+import {
+  SkyLookupShowMoreModalComponent
+} from './lookup-show-more-modal.component';
+
+import {
+  SkyAutocompleteModule
+} from '../autocomplete/autocomplete.module';
+
+import {
+  SkyLookupResourcesModule
+} from '../shared/lookup-resources.module';
 
 @NgModule({
   declarations: [
-    SkyLookupComponent
+    SkyLookupComponent,
+    SkyLookupShowMoreModalComponent
   ],
   imports: [
     CommonModule,
     FormsModule,
     SkyAutocompleteModule,
-    SkyTokensModule
+    SkyDataManagerModule,
+    SkyIconModule,
+    SkyInfiniteScrollModule,
+    SkyLookupResourcesModule,
+    SkyModalModule,
+    SkyRepeaterModule,
+    SkyThemeModule,
+    SkyTokensModule,
+    SkyViewkeeperModule
```

Comment:
We're adding quite a few modules to Lookup's dependencies, including some bulky modules such as Data Manager and Repeater. Might be something to discuss over the phone, just to confirm that's the direction we want to go in.

Reply:
We split things up for this with a new modal provider system.